### DISCUSSION TITLE
How to inject the current_user into a middleware ?

### DISCUSSION THREAD

**Discussion by gikpro** [fastapi-users/fastapi-users#1069] [Q&A]
I am trying to know what user is online without reinventing the wheel using [your current_user](https://github.com/fastapi-users/fastapi-users/blob/master/docs/usage/current-user.md). I must know the current user in all the frontend endpoints that i have created and do some logic with it and passing it in my jinja templates. Thanks !

**Comment by gikpro** [comment]
Sorry the message is duplicate. I can't delete it

**Comment by Dogrtt** [comment]
https://github.com/fastapi-users/fastapi-users/discussions/1079
